portfolio/src/main/java/com/google/sps/servlets/PersonalDetailsServlet.java

```diff
@@ -16,9 +16,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String firstName = "Mark";
     String lastName = "Gacoka";
     int currentAge = 20;
-    String schoolYear = "Sophomore";
+    String[] randomFacts = {"I am 5'9\"", "I was born in Kenya", "I have had multiple near death experience", "I can type upto 170 wpm"};
```

Comment:
I guess with your implementation in previous PR, I was expecting multiple personalDetails (for example: you, your best friend, etc...) - but this is fine as well!